ui/src/test/Header.test.js

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom'
+import Header from '../components/Header/Header';
+import { USER } from './Fixtures/user';
+
+describe("Test Header component", () => {
+    
+    beforeEach(() => {
+        render(<Header user={USER}/>);
+    })
+
+    it("Should have image tag", () => {
+        const imageElement = screen.getByRole('img');
+        expect(imageElement).toBeInTheDocument();
+    })
+
+    it("Should have src tag", () => {
+        const MOCK_IMG_SRC = USER.picture.url;
+        const imageElement = screen.getByRole('img', { 'src': MOCK_IMG_SRC });
+        expect(imageElement).toBeInTheDocument();
+    })
+
+    it("Should have heading username", () => {
```

Comment:
please add the test case for H1 and content of the H1

Reply:
Is this not it?

Reply:
Text is fine but the heading level is required.

`const headingElement = screen.getByRole('heading', { level : 1 });`